nrrd-design-system/components/components/accordions/accordions--mobile-menu.hbs

```diff
@@ -0,0 +1,24 @@
+<main id="foo" class="container-page-wrapper layout-state-pages">
+  <div id="top" aria-label="anchor for mobile navigation" class="mobile-nav-anchor"></div>
+  <div is="mobile-nav" class="mobile-nav sticky stuck">
+    <button is="aria-toggle" aria-controls="mobile-nav-content" aria-expanded="false" type="button">
+      <div class="flex-row">
+        <h3 class="mobile-nav-header flex-row-flex">{{ label }}</h3>
+        <span class="hide-expanded flex-row-icon">
+          <icon class="icon icon-chevron-sm-down"></icon>
```

Comment:
This chevron appears much larger relative to its text in the style guide than it does on the site. Any ideas why @adborden?

## Style guide
![style guide accordion menu](https://user-images.githubusercontent.com/32855580/42479583-72933ed8-838e-11e8-8ee4-962638918d94.png)

## Production site
![site accordion menu](https://user-images.githubusercontent.com/32855580/42479655-c1f1583e-838e-11e8-99d4-db120c05712e.png)